app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Kindly verify that you deployed your app. Your local app is radically different from the deployed app and your local version does not have a splash screen. Kindly inspect that.

  Local version login page | Deployed version login page
  :---:|:---:
  ![klo](https://user-images.githubusercontent.com/54848363/195971745-569aae96-12bd-4875-905a-98914757abd6.jpg) | ![lof](https://user-images.githubusercontent.com/54848363/195971743-92ba53e4-c464-4841-a344-94a944229fe1.jpg)